microshift_troubleshooting/microshift-troubleshoot-backup-restore.adoc

```diff
@@ -0,0 +1,67 @@
+
+[id="troubleshoot-backup-restore-microshift-backup-data-failed"]
+== Backup data failed
+Data backups are automatic on `rpm-ostree` systems. If you are not using an `rpm-ostree` system and attempted to create a manual backup, the following reasons could cause the backup to fail:
+
+* If {product-title} stopped running because of an error, you cannot perform a backup of the data.
+** First, make sure the system is healthy.
+** Then, stop it in a healthy state before attempting a backup.
+** You might need to wait several minutes after a system start to successfully stop {product-title}. Allow the system to complete health checks and any other background processes before attempting to back up.
+
+* If you do not have sufficient storage for the data, the backup fails. Ensure that you have enough storage for the {product-title} data.
+
+* If you do not have sufficient permissions, a backup can fail. Ensure you have the correct user permissions to create a backup and perform the required configurations.
+
+[id="troubleshoot-backup-restore-microshift-backup-logs"]
+== Backup logs
+* Logs print to the console during manual backups.
+* Logs are automatically generated for `rpm-ostree` system automated backups as part of the {product-title} journal logs. You can check the logs by running the following command:
++
+[source, terminal]
+----
+$ sudo journalctl -u microshift
+----
+
+[id="troubleshoot-backup-restore-microshift-restore-data-failed"]
+== Restore data failed
+The restoration of data can fail for a variety of reasons, including storage and permission issues. Mismatched data versions can cause failures when {product-title} restarts.
+
+[id="troubleshoot-backup-restore-microshift-RPM-OSTree-data-restore-failed"]
+=== RPM-OSTree-based systems data restore failed
+Data restorations are automatic on `rpm-ostree` systems, but can fail, for example:
+
+* The only backups that are restored on `rpm-ostree` systems are backups from the current deployment or a rollback deployment. Backups are not taken on an unhealthy system.
+
+** Only the latest backups that have corresponding deployments are retained. Outdated backups that do not have a matching deployment are automatically removed.
+
+** Data is usually not restored from a newer version of {product-title}.
+
+** Ensure that the data you are restoring follows same versioning pattern as the update path. For example, if the destination version of {product-title} is an older version than the version of the {product-title} data you are currently using, the restoration can fail.
+
+[id="troubleshoot-backup-restore-microshift-rpm-manual-restore-data-failed"]
+=== RPM-based manual data restore failed
+If you are using a RPM system that is not `rpm-ostree` and attempted to restore a manual backup, the following reasons could cause the restoration to fail:
+
+* If {product-title} stopped running because of an error, you cannot restore data.
+** First, make sure the system is healthy.
```

Comment:
If the steps on lines 55 and 56 should be taken in a specific order, place them in a numbered list. If you use a numbered list, you can revise the steps to simply "Make sure the system is healthy." and "Start the system in a healthy state before you attempt to restore the data."